.github/workflows/os-check.yml

```diff
@@ -29,8 +29,12 @@ jobs:
           profile: minimal
           toolchain: stable
       - uses: actions/checkout@v2
-      - name: cargo check
+      - name: Create Cargo.lock
         uses: actions-rs/cargo@v1
         with:
-          command: check
+          command: update
```

Comment:
I don't think this explicit update step is needed?

Reply:
Nope, I forgot to remove it .

It's part of the code required for caching.

Reply:
I will open another PR to fix it.

Looks like I need to slow down a bit to ensure quality of the PR/code.

Reply:
Done!
#74 shall fix this.